l10n_ir_country_states/README.rst

```diff
@@ -0,0 +1,42 @@
+Iran Country States
```

Comment:
Any advice on how using screenshots in module description?
As I can see there is an issue using images
[OCA Issue](https://github.com/OCA/maintainer-tools/issues/112)
and
[Odoo Issue](https://github.com/odoo/odoo/issues/8300)